lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -120,8 +122,8 @@ sub call_rpc {
                     if (my $err = $tx->error) {
                         $details .= ', code: ' . ($err->{code} // 'n/a') . ', response: ' . $err->{message};
                     }
-                    warn "WrongResponse [$msg_type], details: $details";
-                    $rpc_failure_cb->($c, $res, $req_storage ) if $rpc_failure_cb;
+                    $log->info("WrongResponse [$msg_type], details: $details");
+                    $rpc_failure_cb->($c, $res, $req_storage) if $rpc_failure_cb;
```

Comment:
I wonder if supplying `$tx->error` (if present) would also be helpful here...

Reply:
Hi @zakame, no sure if I am following correctly, but the `$details` string does already have some attributes from `$tx->error`, precisely `code` and `message`. When you wonder if it would be helpful do you mean to add some other attribute (or all of them) from `$tx->error` or if it will be better to just remove all `$tx->error` information at all?

Reply:
@diego-binary you're not passing `$details` to the callback, though?

The point is that the action may vary depending on what exactly went wrong.